app/src/main/java/io/intrepid/contest/screens/conteststatus/ContestStatusPresenter.java

```diff
@@ -60,16 +59,14 @@ private void showContestStatusScreen(ContestStatusResponse response) {
         if (response.contestStatus.hasContestEnded()) {
             view.showResultsAvailableFragment();
             disposables.clear();
-            return;
+        } else {
+            switch (persistentSettings.getCurrentParticipationType()){
+                case JUDGE:
+                    view.showContestOverviewPage();
+                    break;
+                default: view.showStatusWaitingFragment();
```

Comment:
can you run the formatter on this? I think view.showStatus should be on the next line.